ballerina/client.bal

```diff
@@ -357,760 +358,425 @@ public isolated client class Client {
-        string path = utils:prepareUrl([BASE_PATH, apiVersion]);
-        json res = check self.get(path);
-        return toMapOfStrings(res);
-    }
 
-    # Lists the Limits information for your organization.
-    #
-    # + return - `OrganizationLimits` as map of `Limit` if successful. Else, the occurred `Error`.
-    #
-    # # Deprecated
-    # This function is deprecated as the method signature is altered.
-    # Use the new and improved `getLimits()` function instead.
-    @display {label: "Get Organization Limits"}
-    @deprecated
-    isolated remote function getOrganizationLimits()
-                                                    returns @display {label: "Organization Limits"}
-                                                    map<Limit>|Error {
-        string path = utils:prepareUrl([API_BASE_PATH, LIMITS]);
-        json res = check self.get(path);
-        return toMapOfLimits(res);
-    }
-
-    # Accesses records based on the specified object ID, can be used with external objects.
-    #
-    # + path - Resource path
-    # + return - JSON result if successful else or else `sfdc:Error`
-    #
-    # # Deprecated
-    # This function is deprecated as it returns a json response.
-    # Use the new and improved `getById(string sObject, string id)` function instead.
-    @display {label: "Get Record"}
-    @deprecated
-    isolated remote function getRecord(@display {label: "Resource Path"} string path)
-                                        returns @display {label: "Result"} json|Error {
-        json|http:ClientError response = self.salesforceClient->get(path);
-        if response is json {
-            return response;
+    // Added APIs
+    
+    # Retrieves the list of individual records that have been deleted within the given timespan.
+    #
+    # + sObjectName - SObject reference
+    # + startDate - Start date of the timespan.
+    # + endDate - End date of the timespan.
+    # + return - `DeletedRecordsResult` record if successful or else `error`
+    isolated remote function getDeleted(string sObjectName, time:Civil startDate, time:Civil endDate) 
+        returns DeletedRecordsResult|error {
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS, sObjectName, DELETED]);
+        string finalUrl = utils:addQueryParameters(path,{'start: check time:civilToString(removeDecimalPlaces(startDate)), 
+            end: check time:civilToString(removeDecimalPlaces(endDate))});
+        return check self.salesforceClient->get(finalUrl);
+    }
+
+    # Retrieves the list of individual records that have been deleted within the given timespan.
+    #
+    # + sObjectName - SObject reference
+    # + startDate - Start date of the timespan.
+    # + endDate - End date of the timespan.
+    # + return - `UpdatedRecordsResults` record if successful or else `error`
+    isolated remote function getUpdated(string sObjectName, time:Civil startDate, time:Civil endDate) 
```

Comment:
this to `getUpdatedRecords`?

Reply:
Updated in [update code according to comments](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/311/commits/cd6ae536defa51bd247b84742f1c480c2cf80ab1)